include/tscore/Regex.h

```diff
@@ -50,8 +57,8 @@ class Regex
   ~Regex();
 
 private:
-  pcre *regex;
-  pcre_extra *regex_extra;
+  pcre *regex{nullptr};
```

Comment:
I know it's the same, but it'd be really cool if we could agree on one syntax here... I thought we were using

```
pcre *regex = nullptr;
```

?